.github/workflows/ci.yml

```diff
@@ -28,6 +28,9 @@ jobs:
       - name: Run lint
         uses: reviewdog/action-golangci-lint@v2
 
+      - name: Run govulncheck
+        uses: golang/govulncheck-action@v1
```

Comment:
```suggestion
        uses: golang/govulncheck-action@v1
        with:
           go-version-input: '1.20'
           check-latest: true
           cache: true
           go-package: ./...
```

> The default Go version in golang-govulncheck-action is 1.20.5, and if used as is, it picks up the 1.20.5 vulnerability.
> ( ref: https://twitter.com/catatsuy/status/1680103133282988032)

How about having them check the latest Go version.

https://github.com/actions/setup-go#check-latest-version